Emerald Insight.js

```diff
@@ -345,8 +350,7 @@ var testCases = [
 				"volume": "106",
 				"attachments": [
 					{
-						"title": "Full Text PDF",
-						"mimeType": "application/pdf"
+						"title": "Snapshot"
```

Comment:
I assume whoever updated the test previously had access to the PDF and therefore got a PDF here, so even if you don't this attachment entry and the one below should be reverted.

Reply:
Actually I added those two tests as examples of hyphen breakdowns. I must have done it originally from my university connection so that I got the PDF and I thought it would be best to retain the more general case where we don't get any pdf. Could you confirm you want it reverted?

Reply:
Yes, I think it'd be better to test the more important functionality, even if that means it will fail for people without that.